docs/examples/graphql/mutation.md

```diff
@@ -1,14 +1,14 @@
-require 'appwrite'
+require 'Appwrite'
```

Comment:
I think this should be lower case based on https://github.com/appwrite/playground-for-ruby/blob/main/lib/playground.rb 🧐